marketplace-service/src/main/java/com/axonivy/market/github/service/impl/GHAxonIvyProductRepoServiceImpl.java

```diff
@@ -119,6 +129,36 @@ public void extractReadMeFileFromContents(Product product, List<GHContent> conte
     }
   }
 
+  @Override
+  public void updateSetupPartForProductModuleContent(Product product,
+      Map<String, Map<String, String>> moduleContents, String tag) throws IOException {
+    if (!NonStandardProduct.isMsGraphProduct(product.getId())) {
```

Comment:
there are many other connectors that can use this advanced build infrastructure and did so in the past? this does not solve the problem at hand @tvtphuc-axonivy